src/registry.js

```diff
@@ -0,0 +1,154 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
```

Comment:
This module doesn't really fit anywhere else then in the `src/` root I think.

Maybe we should move `plot_api/plot_config.js` to the `src/` root too. That way both modules with objects that can be modified at runtime would be in a very up-front location.